### ISSUE TITLE
Breakopint tree does not show scrollbar when number of breakpoints < max and it overflows

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#39796]
Details in https://github.com/Microsoft/vscode/issues/25384#issuecomment-349655644

To repro

1. Have 5 breakpoints
2. Resize the vscode window such that the breakpoints view does not have enough space -> the scroll bar is not shown

**Comment by isidorn** [comment]
From my investigation, the issue here is that the split views do not seem to take into account if there is not enough space for them to layout.
This all starts from [here](https://github.com/Microsoft/vscode/blob/master/src/vs/base/browser/ui/splitview/splitview.ts#L373) where this `leyaoutViews` does not take into account the passed sizes.

Assignign @joaomoreno and @sandy081 since I am not sure who is the owner

**Comment by joaomoreno** [comment]
@isidorn You tell the splitview: _I want the breakpoints view to be at least X pixels tall and at most X pixels tall_. So, the view will be exactly X pixels tall. The splitview can't resize your breakpoints view beyond X pixels, because it has a minimum size of X pixels.

Unfortunately there is no good solution for this. You want a fixed size view, unless there is no space for it, at which point it should start collapsing. The splitview doesn't support this. At best, the splitview can inform you when either its size or its content size change and you can use those two values to determine whether there is enough space for your breakpoints view to be fixed size.

**Comment by isidorn** [comment]
I looked into making the breakpoints view the regular split view.
However after using it for some time it felt inferiour to our current solution. Thus I have decided to live with the bugs of the current implenetation and not update the view to be resizable.

here's one sitaution where the new solution could hurt users
Closing as will not fix
![views](https://user-images.githubusercontent.com/1926584/33937118-81cb99f2-e002-11e7-9c36-609f5fcba8d8.gif)
